soh/src/overlays/actors/ovl_Bg_Ice_Shelter/z_bg_ice_shelter.c

```diff
@@ -320,6 +357,24 @@ void func_8089107C(BgIceShelter* this, GlobalContext* globalCtx) {
     CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->cylinder1.base);
 }
 
+// Check for an Ice Arrow hit (for "Blue Fire Arrows" enhancement). Copied from default blue fire check
+void CheckIceArrowHit(BgIceShelter* this, ColliderCylinder cylinder, s16 type, GlobalContext* globalCtx) {
```

Comment:
does this naming make sense? i'd think `CheckIceArrowHit` would be a `bool` that returns `true` if the ice arrow hit and `false` if it didn't

it seems this is doing more than just "checking" if the ice arrow hit

i'm not sure what a better name would be, but maybe updating the comment to explain what this is doing in the case of an ice arrow check

or maybe just rename it `MeltIfBlueFireArrowHit` or something (just throwing a name idea out there, i'm not sold on `MeltIfBlueFireArrowHit` either)

Reply:
Good point!  I will revisit the name/description here

Reply:
Clarified the function name and comment here